README.md

```diff
@@ -204,6 +212,7 @@ The `tuned` package is required for the default provider.
         value: 0
       - name: /sys/kernel/debug/x86/ibrs_enabled
         value: 0
+    kernel_settings_systemd_cpu_affinity: "1,3,5,7"
```

Comment:
ditto (above you say 'space delimited', but here you show a comma-delimited list.)